src/Test/Plutip/Internal/LocalCluster.hs

```diff
@@ -234,31 +242,38 @@ checkProcessesAvailable requiredProcesses = do
         <> show (catMaybes results)
 
 waitForRelayNode :: Tracer IO TestsLog -> RunningNode -> IO ()
-waitForRelayNode trCluster rn = do
-  liftIO $ recoverAll policy (const getTip)
+waitForRelayNode trCluster rn =
+  liftIO $ do
+    recoverAll policy wait
   where
-    policy = constantDelay 500000 <> limitRetries 5
-    getTip = trace >> void (Tools.queryTip rn)
+    policy = constantDelay 500000 <> limitRetries 50
```

Comment:
Consider extracting into some kind of Config?

Reply:
Made an issue for that: #125